app/src/main/java/com/pppp/s/main/view/custom/SimpleGridLayoutManager.kt

```diff
@@ -0,0 +1,9 @@
+package com.pppp.s.main.view.custom
+
+import android.content.Context
+import android.support.v7.widget.GridLayoutManager
+
+class SimpleGridLayoutManager(
```

Comment:
I think this class is useless and its usage can be replaced with:
`GridLayoutManager(context, 3)`